### ISSUE TITLE
Classes seems to have no effect

### ISSUE THREAD

**Issue by valdisiljuconoks** [arnedesmedt/vue-ads-table-tree#22]
Hi,

I'm trying to style header cells. Seems like following class has no effect over final markup (no additional classes are added apart from default ones).

```
classes: {
    '0': {
        'bg-dark-gray': true
    }
},
```

**Comment by valdisiljuconoks** [comment]
hi, what about this one? thx

**Comment by pavelav** [comment]
Can't get styling to work either. By default, odd/even rows should have alternate background colors, but that doesn't work too. Same applies to the [demo app](https://jsfiddle.net/arnedesmedt/7my8L42q). Tried latest Chrome (76.0.3809.100) and Firefox (68.0.1) on Linux. @arnedesmedt, please take a look. Thank you!

**Comment by pavelav** [comment]
Regarding default behavior: among classes referenced in `defaultClasses.js`, the following are not defined in `vue-ads-table-tree.css`:
```css
vue-ads-bg-grey-lighter
vue-ads-bg-grey-lightest
```
which is why it doesn't work.

**Comment by arnedesmedt** [comment]
changes are available in v2.3.7
